utils/layer.py

```diff
@@ -52,16 +52,20 @@ def add_attributes(layer, assets):
     layer.updateFields()
 
 
-def add_features(layer, list_of_assets):
+def add_features(layer, list_of_assets, dockwidget):
```

Comment:
I think we should have a solution without the ``dockwidget`` to make this simpler.